social_protection/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from core import models as core_models
+from policyholder.models import PolicyHolder
+
+
+class BenefitPlan(core_models.HistoryBusinessModel):
+    code = models.CharField(db_column='Code', max_length=8, null=False)
+    name = models.CharField(db_column='NameBF', max_length=255, null=False)
+    date_from = models.DateTimeField(db_column="DateFrom")
+    date_to = models.DateTimeField(db_column="DateTo")
```

Comment:
I think that those fields are redundant @jdolkowski . Such time scope is usually handled by HistoryBussinessModel: date_valid_from and date_valid_to.
@dborowiecki  @malinowskikam  What do you think about this?

Reply:
It makes sense. I think we should remove those fields.

Reply:
I asked @malinowskikam, he also agrees.

Reply:
done

Reply:
I think it has to be removed. HistoryBusinessModel already covers the use case of defining expiring models.